### PR TITLE
Notification Likes list: show user profile in bottom sheet when user tapped.

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref #16244 

In the user list for a Like Notification, a user profile bottom sheet is shown when the user row is tapped. The view currently only shows:
- The "user information" block - gravatar, name, username, and a stub for the user bio.
- Section header for the site section.

The bottom sheet should render correctly on iPhone in portrait and landscape.

To be addressed in follow-up PRs:
- Correctly placing the sheet on iPad.
- Adding a table row for the user's site (to be displayed in the currently empty site section).

To test:
- Enable the `newLikeNotifications` feature.
- Run on an iPhone. (It will show on an iPad, just not in the right place.)
- Go to Notifications > Likes and tap a notification.
- Tap a user.
  - Verify the bottom sheet appears as described above.
  - Verify the sheet is partial height in portrait.
  - Verify the sheet is full height in landscape.
- Disable the `newLikeNotifications` feature.
- Go to Notifications > Likes and tap a notification.
  - Verify tapping the user opens the user's site.

| light_portrait | dark_portrait |
|--------|-------|
| light_landscape | dark_landscape |

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by ScoutHarris** [comment]
Hey @mattmiklic . This is still a WIP, but would you mind reviewing what I have so far? There weren't any specs in the User Profiles design (pbArwn-K3-p2), so I winged it. Thank you!

**Review by mattmiklic — CHANGES_REQUESTED** [review]
This is really close! Just a few adjustments:

- 5pt more spacing between the top of the sheet and the avatar/name
- Increase avatar size to 56pt
- User name is the Title 3 text size, New York font, Medium weight

The rest is right on the money!

**Comment by ScoutHarris** [comment]
Thanks @mattmiklic . Changes made and screenshots updated in the description.

**Review by mattmiklic — APPROVED** [review]
Sorry my mistake -- the user's name should be semibold, rather than medium. Otherwise this looks great now!

**Comment by ScoutHarris** [comment]
And done! Thanks @mattmiklic !

File: WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+// MARK: - DrawerPresentable Extension
+
+extension UserProfileSheetViewController: DrawerPresentable {
+
+    var collapsedHeight: DrawerHeight {
+        if traitCollection.verticalSizeClass == .compact {
+            return .maxHeight
+        } else {
+            return .contentHeight(300)
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift]: Super nitpicky, and probably just a question of preference, but if you like we could do away with the else clause and just have the `return .contentHeight(300)`.

**Review by aerych — APPROVED** [review]
Howdy @ScoutHarris 👋 
Tested and everything looked good.  I left one suggestion but feel free to ignore it. :) 
I've some concern about the number of todos, but I get that these will be addressed in follow up PRs, so fine to roll with it. :)
:shipit: from me.

File: WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift (hunk 1 above)
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/User Profile Sheet/UserProfileSheetViewController.swift]: Busted copying & pasting! 😄 Thank you!

**Pull request merged**